services/QuillLMS/app/services/complete_account_creation.rb

```diff
@@ -8,6 +8,11 @@ def initialize(user, ip)
   end
 
   def call
+    if user.admin?
+      AdminInfo.create(admin_id: user.id, approval_status: AdminInfo::SKIPPED) # setting approval status to SKIPPED to ensure that self-created admins who exit or bypass the school verification step will have to complete it eventually
```

Comment:
I'd be tempted to make this a function call on the `User` model sort of like `verify_email` so that the service doesn't have to understand the table relationships between User and AdminInfo, but that's a stylistic preference and I think this works fine given how few places this needs to be called.